manim/mobject/geometry/polygram.py

```diff
@@ -472,7 +472,6 @@ class Star(Polygon):
     Examples
     --------
     .. manim:: StarExample
-        :save_as_gif:
```

Comment:
Gif not working anymore?

Reply:
The documentation build indeed failed on this example (and thus likely is broken in general, we need to investigate rendering with `--format=gif`), but I also see no good reason for this singular example being rendered as a gif while every other example in our docs is a video (or still image).

Reply:
I tried to fix GIF but still have some issue:

![image](https://github.com/ManimCommunity/manim/assets/27275099/6da30d89-d090-4e63-b4a5-a02e87b5439f)

(from `output_container.mux`).